mir_eval/separation.py

```diff
@@ -442,6 +443,102 @@ def bss_eval_images(reference_sources, estimated_sources,
         return (sdr, isr, sir, sar, popt)
 
 
+def bss_eval_images_framewise(reference_sources, estimated_sources,
+                              window=30*44100, hop=15*44100,
+                              compute_permutation=False):
+    """Framewise computation of bss_eval_images
+
```

Comment:
hm, that might be better, yes...

it's

```
Emmanuel Vincent, Shoko Araki, Fabian J. Theis, Guido Nolte, Pau Bofill,
Hiroshi Sawada, Alexey Ozerov, B. Vikrham Gowreesunker, Dominik Lutter
and Ngoc Q.K. Duong, "The Signal Separation Evaluation Campaign
(2007-2010): Achievements and remaining challenges", Signal Processing,
92, pp. 1928-1936, 2012.
Emmanuel Vincent, Hiroshi Sawada, Pau Bofill, Shoji Makino and Justinian
P. Rosca, "First stereo audio source separation evaluation campaign:
data, algorithms and results," In Proc. Int. Conf. on Independent
Component Analysis and Blind Source Separation (ICA), pp. 552-559, 2007.
```
but maybe the 1st reference is sufficient